src/DotTiled.Example/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using DotTiled.Serialization;
+
+namespace DotTiled.Example;
+
+class Program
+{
+  static void Main(string[] args)
+  {
+    Quick();
+    Manual();
+  }
+
+  // QUICK START
+  // Automatic and easy way to load tilemaps.
+  static void Quick()
+  {
+    var loader = Loader.Default();
+    var map = loader.LoadMap("tilemap.tmx");
+
+    // You can do stuff with it like...
+    Console.WriteLine($"Tile width and height: {map.TileWidth}x{map.TileHeight}");
+    TileLayer layer0 = (TileLayer)map.Layers[0]; // Get a layer
+    Console.WriteLine($"Tile in layer 0 at 0, 0: {layer0.Data.Value.GlobalTileIDs.Value[0]}");
+  }
+
+  // MANUAL
+  // Manually load a map, if you need to load from a custom source
+  static void Manual()
+  {
+    using var mapFileReader = new StreamReader("tilemap.tmx");
+    var mapString = mapFileReader.ReadToEnd();
+    using var mapReader = new MapReader(mapString, ResolveTileset, ResolveTemplate, ResolveCustomType);
+    var map = mapReader.ReadMap();
+
+    // Now do some other stuff with it...
+    StringProperty hello = (StringProperty)map.Properties.FirstOrDefault(property => property.Name == "hello");
+    Console.WriteLine($"Layer 1 name: {map.Layers[0].Name}");
+    Console.WriteLine($"Property 'hello': {hello.Value}");
+
+    // Now with tileset
+    Tileset tileset = map.Tilesets[0];
+    Console.WriteLine($"Tileset 0 source: {tileset.Source.Value}");
+    Console.WriteLine($"Tileset 0 image 0 source: {tileset.Image.Value.Source.Value}");
+  }
+
+  /* This function is responsible for loading all tilesets required by a tilemap, if you
+   want to use a custom source. */
```

Comment:
Prefer single line comment:
```suggestion
  // This function is responsible for loading all tilesets required by a tilemap, if you
  // want to use a custom source.
```